toolkits/nature/packages/nature-section-heading/view/section-heading.hbs

```diff
@@ -1,14 +1,24 @@
-<div class="c-section-heading {{no-bt}}">
-	{{#if url}}
-	<h2>
-		<a href="{{url}}" class="c-section-heading__link">
+<div class="c-section-heading {{#if no-bt}}{{no-bt}}{{/if}}">
```

Comment:
I think with handlebars you don't need to check the expression exists, if it doesn't it won't render anything.
Although I guess it makes it clear this way.

Reply:
I think it's better to be clear and explicit here. I'm not sure if that functionality is true for older versions of handlebars, and some people will be less familiar with the syntax.